### DISCUSSION TITLE
VS Code extension for config file syntax highlighting

### DISCUSSION THREAD

**Discussion by blaylockbk** [dtcenter/METplus#1243] [Tips/Tricks]
Not a question, just sharing...

I'm not sure how many out there use VS Code, but I made a little extension to apply syntax highlighting to MET configuration files and wanted to share it. It's not perfect, but it does help me visualize what I'm doing when editing a config file.

Install the extension from the marketplace: 
https://marketplace.visualstudio.com/items?itemName=brian-blaylock.met-syntax

Here is an example
![compare](https://user-images.githubusercontent.com/6249613/140621023-c5c669e4-0a77-4ff6-8d8e-86c607b90846.png)

Anyways, I wanted to share this for the other VS Code users. I'm not an experienced extension author (I literally did this after watching a YouTube tutorial and following some docs), so if anyone has other ideas to make it better, feel free to submit pull requests. There is a lot of potential with an extension like this to help a person use MET.

**Comment by JohnHalleyGotway** [comment]
Hi @blaylockbk, thanks for the recommendation about syntax highlighting of MET config files!

Our support team talked about your post internally and decided to repurpose the existing "General" category into one named "General Discussion". And we switched it from Question/Answer to an Open Discussion format. And then moved your discussion there.

For posts to the other Question/Answer categories, once the initial question has been answered, we mark it as such and "Lock" it to prevent future posts. Since your discussion doesn't fit that paradigm, we wanted a way to keep it open.

Hopefully other users will feel free to add other syntax highlighting suggestions as well.

Thanks!
